src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -241,15 +241,20 @@ const TeamMembers = createReactClass({
           <div style={{textTransform: 'none'}}>{this.renderDropdown(access)}</div>
         </PanelHeader>
         {this.state.teamMemberList.length ? (
-          this.state.teamMemberList.map((member, i) => (
-            <StyledMemberContainer key={i}>
-              <IdBadge avatarSize={36} user={member} useLink orgId={params.orgId} />
+          this.state.teamMemberList.map(member => (
+            <StyledMemberContainer key={member.id}>
+              <IdBadge
+                avatarSize={36}
+                user={member.user || member}
```

Comment:
`member.user` is where the avatar is stored.

Reply:
what's the reason for `|| member`?

Reply:
It's possible to invite members that are not registered in sentry to a team (i.e. `member.user` is null). In this case it will use the invited member's email (maybe later on we can add some visual design to the badge to indicate they are "invited" and not registered).